examples/custom_action/example_server/swagger_server/models/request.py

```diff
@@ -0,0 +1,215 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from datetime import date, datetime  # noqa: F401
```

Comment:
The F401s are still remaining!

Reply:
I think these are unused imports, we can choose to remove them.

Reply:
done